rll/collectd.sh

```diff
@@ -259,11 +263,22 @@ else
   echo "swapfile not setup, skipping collectd swap plugin"
 fi
 
+# Populate RS_RLL_PORT
+source /var/run/rightlink/secret
+/usr/local/bin/rsc --rl10 rl10 put_hostname /rll/tss/hostname hostname=$COLLECTD_SERVER
```

Comment:
This is a little deceptive. the secret file isn't source for rsc, its sourced for writing the collectd config 3-4 lines below this.